### PR TITLE
Torso

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

**Comment by NadiaHG** [comment]
Tested on cob4-2 with the actual ros_canopen release version and ros_controllers from ipa320 and branch indigo-devel

File: cob_bringup/robots/cob4-2.xml [hunk 1]
```diff
@@ -133,6 +133,16 @@
...
+		<include file="$(find cob_bringup)/drivers/canopen_402.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="component_name" value="torso" />
+			<arg name="can_module" value="can0" />
+		</include>
+		<include file="$(find cob_bringup)/controllers/generic_controller.launch">
+			<arg name="robot" value="$(arg robot)"/>
+			<arg name="component_name" value="torso"/>
+		</include>
+
```
Comment (fmessmer) [inline cob_bringup/robots/cob4-2.xml:145]: in [L96](https://github.com/ipa320/cob_robots/pull/316/files#diff-6d681f7484f0edd03a9af42bf5655241L96) this block is already include....just uncomment it there....

only change `can1` has to be changed to `can0` there!

**Comment by fmessmer** [comment]
superseded by #318?

**Comment by fmessmer** [comment]
Yes, superseded by #318

**Pull request closed without merging**